packages/demo-saas/src/index.tsx

```diff
@@ -1,18 +1,10 @@
-import React from 'react';
+import React, { StrictMode } from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter } from 'react-router-dom';
-import { FronteggProvider } from '@frontegg/react';
 import { App } from './App';
 
 ReactDOM.render(
-  <BrowserRouter>
-    <FronteggProvider
-      contextOptions={{
-        baseUrl: `https://david.frontegg.com`,
-      }}
-    >
-      <App />
-    </FronteggProvider>
-  </BrowserRouter>,
```

Comment:
maybe we should keep `BrowserRouter` for testing?

Reply:
yup we have it in App.tsx